includes/wp-admin/class-blog-settings-fields.php

```diff
@@ -295,11 +303,26 @@ class="large-text"
 			><?php echo esc_textarea( implode( PHP_EOL, (array) $also_known_as ) ); ?></textarea>
 		</label>
 		<p class="description">
-			<?php esc_html_e( 'If you’re moving from another account to this one, you’ll need to create an alias here first before transferring your followers. This step is safe, reversible, and doesn’t affect anything on its own. The migration itself is initiated from your old account.', 'activitypub' ); ?>
+			<?php esc_html_e( 'If you&#8217;re moving from another account to this one, you&#8217;ll need to create an alias here first before transferring your followers. This step is safe, reversible, and doesn’t affect anything on its own. The migration itself is initiated from your old account.', 'activitypub' ); ?>
 		</p>
 		<p class="description">
 		<?php echo \wp_kses_post( \__( 'Enter one account per line. Profile links or usernames like <code>@username@example.com</code> are accepted and will be automatically normalized to the correct format.', 'activitypub' ) ); ?>
 		</p>
 		<?php
 	}
+
+	/**
+	 * Hide Social Graph field callback.
+	 */
+	public static function hide_followers_callback() {
+		?>
+		<label>
+			<input type="checkbox" name="activitypub_hide_social_graph" id="activitypub_hide_social_graph" value="1" <?php \checked( '1', \get_option( 'activitypub_hide_social_graph', '0' ) ); ?> />
+			<?php \esc_html_e( 'Hide Followers and Following on Profile', 'activitypub' ); ?>
+		</label>
+		<p class="description">
+			<?php esc_html_e( 'By default, these lists are public. Hiding them removes them from your profile, but people you follow will still see that you follow them.', 'activitypub' ); ?>
```

Comment:
```suggestion
			<?php esc_html_e( 'People you follow will still see that you follow them.', 'activitypub' ); ?>
```
Sorry, missed that one.